python/pyiceberg/transforms.py

```diff
@@ -237,7 +282,7 @@ class TimeResolution(IntEnum):
     SECOND = 0
 
 
-class TimeTransform(Transform[S, int], Singleton):
+class DatesTransform(Transform[S, int], Singleton):
```

Comment:
I changed this one to match with the Java implementation, but not really required.